tools/pre-commit.sh

```diff
@@ -7,44 +7,13 @@
 # To do this, execute, from repo root
 # ln -s $(pwd)/tools/pre-commit.sh $(pwd)/.git/hooks/pre-commit
 
-rm -f /tmp/check_apache_header
-rm -f /tmp/check_java_package
-RESULT=0
+bazel query 'kind(java_*, //...) - kind(java_proto_library, //...) - //third_party/... - attr("tags", "checkstyle_ignore", //...)' | sort >/tmp/all_java_targets
+bazel query 'kind(checkstyle_test, //...)' | sed -e 's|-checkstyle||g' | sort >/tmp/checkstyle_covered_java_targets
 
-check_apache_header() {
-  HEADER="Copyright $(date +'%Y') The StartupOS Authors."
-  if grep -qL "$HEADER" $1; then
-    echo -n # Found header, things are OK
-  else
-    echo "$(tput setaf 1) License header NOT found in $1 $(tput sgr0)"
-    touch /tmp/check_apache_header
-  fi
-}
+NON_COVERED_JAVA_TARGETS=$(comm -23 /tmp/all_java_targets /tmp/checkstyle_covered_java_targets)
 
-check_java_package() {
```

Comment:
Are we still checking the Java package?

Reply:
@oferb I replaced it with checkstyle check, see [here](https://github.com/google/startup-os/blob/integrate-checkstyle/tools/checkstyle/config.xml#L111-L114)